.travis.yml

```diff
@@ -19,9 +18,10 @@ install:
 
 before_script:
   - ct-test-srv -config ./test/config/cttestsrv.json &
```

Comment:
I think you need to `chmod` the test password file to pass your own filesystem permissions check (Git can't retain the mode on a file in the repo can it?)

Right now CI is failing with:
> 2019/02/21 20:35:38 Unable to create ct-woodpecker: permissions 664 for password file "test/config/db_password" are too open